services/121-service/src/scripts/factories/registration-seed-factory.ts

```diff
@@ -73,6 +77,13 @@ export class RegistrationSeedFactory extends BaseSeedFactory<RegistrationEntity>
       newRegistrationIds,
       programId,
     );
+    if (includeEvents) {
```

Comment:
Nitpick: Instead of passing this param to this deeper factory does it not make more sense to call the registrationeventDataFactory from MockSeedFactoryService and have this decision live there?

Reply:
Hmm, this is some time ago already, but I believe I had that first, but that didn't work. The reason is that I use the just-created registrationIds to match for the event data. Following the existing pattern for registration-attribute-data right above.